Plugin/Block/Product/Image.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Copyright © Fisheye Media Ltd. All rights reserved.
+ * See LICENCE.txt for licence details.
+ */
+namespace Fisheye\Lazyload\Plugin\Block\Product;
+
+
+class Image
+{
+
+    /**
+     * Plugin for lazyloading images - Set template to Fisheye_Lazyload
+     *
+     * @param \Magento\Catalog\Block\Product\Image $subject
```

Comment:
- [ ] Move `Magento\Catalog\Block\Product\Image` to `use` statement once class name changed